bundle/wavefrontcollectors.wavefront.com.crd.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
These seem similar to the crds under `deploy/crds`. Are these required? Can we delete these two bundle/ files?

Reply:
Done.